index.bs

```diff
@@ -130,6 +130,12 @@ The data provided by the {{OrientationSensor}} subclasses are similar to data fr
    motion sensors are used to obtain the orientation data, thus obviating possible interoperability issues.
 1. Instances of {{OrientationSensor}} subclasses are configurable via {{SensorOptions}} constructor parameter.
 
+Use Cases and Requirements {#usecases-requirements}
+==============================
+
+This document attempts to address the <cite><a href="https://w3c.github.io/motion-sensors/">Use Cases and
```

Comment:
This document attempts to address the use cases and requirements documented in Motion Sensors Explainer.

(Use the right name of the doc you link to for consistency.)